src/resolve/resolved-mdns.c

```diff
@@ -35,38 +35,6 @@ void manager_mdns_stop(Manager *m) {
         m->mdns_ipv6_fd = safe_close(m->mdns_ipv6_fd);
 }
 
-int manager_mdns_start(Manager *m) {
-        int r;
-
-        assert(m);
-
-        if (m->mdns_support == RESOLVE_SUPPORT_NO)
```

Comment:
This leaves `mdns_support` completely unused. That field would have to be removed too.